GOAT-Compiler/RecSymbolTable.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GOAT_Compiler
+{
+    class RecSymbolTable : ISymbolTable
+    {
+
+        private Table tables;
+        private Table mainTable;
+        private bool buildFlag = false;
```

Comment:
This seems poorly named. As i read it,  if "buildflag" is false, it is building, which seems wrong